commit 8fd0b52f20e28759a88e1202716f60e773b11dd3
Author: Jaroslav Šmíd <jardasmid@gmail.com>
Date:   Mon Jul 9 19:12:42 2012 -0700

    Backport: Bug 51375: Xorg doesn't set status for RRGetOutputInfo

    https://bugs.freedesktop.org/show_bug.cgi?id=51375
    https://bugs.freedesktop.org/attachment.cgi?id=63397

    Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
    Reviewed-by: Keith Packard <keithp@keithp.com>
    Tested-by: Daniel Stone <daniel@fooishbar.org>

    Backported from Arctica GH 3.6.x branch.

    v2: backport to nx-libs 3.6.x (Ulrich Sibiller)
    v3: backport to nx-libs 3.5.0.x (Mihai Moldovan)

diff --git a/nx-X11/programs/Xserver/randr/rroutput.c b/nx-X11/programs/Xserver/randr/rroutput.c
index 1ecde31..693f5a5 100644
--- a/nx-X11/programs/Xserver/randr/rroutput.c
+++ b/nx-X11/programs/Xserver/randr/rroutput.c
@@ -456,6 +456,7 @@ ProcRRGetOutputInfo (ClientPtr client)
     pScrPriv = rrGetScrPriv(pScreen);
 
     rep.type = X_Reply;
+    rep.status = RRSetConfigSuccess;
     rep.sequenceNumber = client->sequence;
     rep.length = OutputInfoExtra >> 2;
     rep.timestamp = pScrPriv->lastSetTime.milliseconds;
